src/utils/client.js

```diff
@@ -16,11 +16,12 @@ const error = require('../storage/logs').error('medmorph-backend:client');
 async function connectToServer(url) {
   // Generate the client_assertion jwt
   const tokenEndpoint = await getTokenEndpoint(url);
-  const clientId = servers.getServerByUrl(url).clientId;
+
+  const { clientId, secret, customScopes } = servers.getServerByUrl(url);
```

Comment:
Would you mind adding the `secret` and `customScopes` to the `servers` collection headers in `src/ui/components/Collections/Collections.jsx` so they can be edited from the ui?